webpack.test.config.js

```diff
@@ -1,65 +1,12 @@
-var webpack = require( 'webpack' ),
+const webpack = require( 'webpack' ),
 	path = require( 'path' ),
-	nodeExternals = require( 'webpack-node-externals' );
+	nodeExternals = require( 'webpack-node-externals' )
+	webpackConfig = require( './webpack.config' );
 
-module.exports = {
-	cache: true,
-	module: {
-		loaders: [
-			{
-				test: /\.json$/,
-				loader: 'json-loader'
-			},
-			{
-				test: /\.html$/,
-				loader: 'html-loader'
-			},
-			{
-				test: /\.scss$/,
-				loader: 'noop-loader',
-			},
-			{
-				test: /\.jsx?$/,
-				loader: 'babel-loader',
-				query: {
-					cacheDirectory: true,
-					presets: [
-						'es2015',
-						'stage-1',
-						'react'
-					],
-					plugins: [
-						"add-module-exports",
-					],
-					babelrc: false,
-				}
-			}
-		]
-	},
-	resolve: {
-		alias: {
-			'react': path.join( __dirname, 'node_modules', 'react' ),
-			'react-dom': path.join( __dirname, 'node_modules', 'react-dom' ),
-			'redux': path.join( __dirname, 'node_modules', 'redux' ),
-			'lib/mixins/i18n': path.join( __dirname, 'client', 'lib', 'mixins', 'i18n' )
-		},
-		extensions: [ '', '.json', '.js', '.jsx' ],
-		root: [
-			path.join( __dirname, 'client' ),
-			path.join( __dirname, 'node_modules' ),
-			path.join( __dirname, 'node_modules', 'wp-calypso', 'client' )
-		],
-		fallback: [
-			path.join( __dirname, 'node_modules', 'wp-calypso', 'node_modules' )
-		]
-	},
-	resolveLoader: {
-		modulesDirectories: [ __dirname + '/node_modules' ]
-	},
-	target: 'node',
-	externals: [
-		nodeExternals( {
-			whitelist: [ 'wp-calypso' ]
-		} )
-	]
-};
\ No newline at end of file
+webpackConfig.externals = [
```

Comment:
Are you sure `webpack.test.config.js` is only loaded in a test? Perhaps it's loaded but not used.

Either way, editing a module like this is a terrible idea:

```
// mod.js
module.exports = { test: 'hello' };

//mod1.js
const mod = require( './mod' );
mod.test2 = 'hi';
module.exports = mod;

//index.js
const mod = require( './mod' );
const mod1 = require( './mod1' );
console.log( 'mod' );
console.dir( mod );
console.log( 'mod1' );
console.dir( mod1 );

//output
PS> node .\index.js
mod
{ test: 'hello', test2: 'hi' }
mod1
{ test: 'hello', test2: 'hi' }

```